libFMS.F90

```diff
@@ -73,18 +73,13 @@ module fms
 
   !> amip_interp
   use amip_interp_mod, only: fms_amip_interp_init         => amip_interp_init, &
-                             fms_amip_interp_get_amip_sst             => get_amip_sst, &
-                             fms_amip_interp_get_amip_ice             => get_amip_ice, &
+                             fms_amip_interp_get_amip_sst => get_amip_sst, &
+                             fms_amip_interp_get_amip_ice => get_amip_ice, &
                              fms_amip_interp_new          => amip_interp_new, &
                              fms_amip_interp_del          => amip_interp_del, &
-                             FmsAmipInterp_type => amip_interp_type, &
-                             assignment(=), &
-                             fms_amip_interp_i_sst        => i_sst, &
```

Comment:
They were removed because I made them private in the first iteration of this PR, and then I neglected to add them back in. `libFMS.F90` has been restored to its original state in 58923c69613ad497d58d16391a2b1db8fd43835e.